.github/workflows/tests.yml

```
@@ -0,0 +1,41 @@
+name: UnitTests
+on: 
+  pull_request:
+    branch: 
+      - '*'
+  push:
+    branch:
+      - '*'
+
+jobs:
+  rpmlinter:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: rpmlinter
+      run:  make spec && docker run -v "$PWD"/python-boto.spec:/specs/python-boto.spec:z ubun/rpmlinter
+
+  python_test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7,3.6]
+    env:
+      BOTO_CONFIG: '/tmp/nowhere'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+      env:
+        AGENT_TOOLSDIRECTORY: '/opt/hostedtoolcache'
+    - name: Install Python ${{ matrix.python-version }} Dependencies    
+      run: |
+        mkdir ~/.ssh && chmod 0700 ~/.ssh && echo "$SSH_KNOWN_HOSTS" >> ~/.ssh/known_hosts && chmod 644 ~/.ssh/known_hosts &&
+        if [ ${{ matrix.python-version }} == 2.7 ]; then python -m pip install -r requirements-py26.txt; fi
+        python -m pip install -r requirements.txt
```

вот это давай разобьем по логическим шагам. вполне вероятно, что в будущем где-то какие-то шаги будут не нужны и будет выглядеть понятно, где и что делается (скип шага удобно отображается в прогоне actions):
```suggestion
    - name: Create .ssh directory for ssh test
      run: |
        mkdir ~/.ssh
        chmod 0700 ~/.ssh
        echo "$SSH_KNOWN_HOSTS" >> ~/.ssh/known_hosts
        chmod 644 ~/.ssh/known_hosts

    - name: Install python ${{ matrix.python-version }} specific dependencies
      run: python -m pip install -r requirements-py27.txt
      if: ${{ matrix.python-version == '2.7' }}

    - name: Install python ${{ matrix.python-version }} dependencies
      run: python -m pip install -r requirements.txt
```

---

Ну и в дополнение. Наверное, надо переименовать py26 файл с зависимостями в py27?
И удалить travis файл?